### PR TITLE
build: Add workflows for bump PRs

### PR DESCRIPTION
**Pull request by gracedo**
- Add labeler workflow to label PRs with the service that is being updated.
- Add make target to run script to regenerate the .github/service-labeler.yaml file with the latest services.
- Add workflow to automatically update the .github/service-labeler.yaml file on every PR if necessary (if new service is added).
- Add step to labeler workflow to label automated bump PRs with `ok-to-test`, and `open-kommander-pr` only for those indicated in .github/bump-pr-labeler.yaml.
- Add automerge workflow to automatically merge automated bump PRs once the required checks are green (and there are no `do-not-merge` labels) -- example https://github.com/gracedo/kommander-applications/pull/8

**What problem does this PR solve?**:
Adding some workflows to better filter PRs by the apps being updated. Mainly, filter bump prs by app. So that we can close out outdated PRs (one day we can automate this also, but labeling is likely the first step).

**Which issue(s) does this PR fix?**:

https://d2iq.atlassian.net/browse/D2IQ-93453

**Special notes for your reviewer**:

I did some initial testing on my fork, but to fully test the new workflows here, we will need to merge this PR first

**Does this PR introduce a user-facing change?**:

```release-note

```

**If the PR adds a version bump, does it add a breaking change in License**:

- [ ] No License Change (or NA).

### CODE REVIEW

File: .github/workflows/automerge.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  automerge:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Enable automerge on automated chart bump PRs
+        id: automerge
+        uses: pascalgn/automerge-action@v0.15.3
+        if: startsWith(github.head_ref, 'chartbump/')
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          MERGE_LABELS: "!do-not-merge,!do-not-merge/testing"
```
Comment (gracedo) [inline .github/workflows/automerge.yaml:33]: ❓ do we want to add `ready-for-review` as a required label for automerging? (automerge would be blocked if that label is missing even if all other checks, including number of approvals, is green)

Comment (cwyl02) [reply .github/workflows/automerge.yaml:33]: I vote yes

Comment (gracedo) [reply .github/workflows/automerge.yaml:33]: we can always change it if we think the human action to add this label is slowing things down 😆 

```suggestion
          MERGE_LABELS: "ready-for-review,!do-not-merge,!do-not-merge/testing"
```

File: make/workflows.mk [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.PHONY: workflow-labeler-yaml-update
+workflow-labeler-yaml-update: ## Updates .github/service-labeler.yaml for use with labeler GH action
+workflow-labeler-yaml-update: ; $(info $(M) updating .github/service-labeler.yaml with latest services)
+	ls -d services/* | awk 'NR > 1' | sed p | awk 'NR % 2 { print $$0 ":" } !(NR % 2) {print "  - " $$0 "/**";}' > .github/service-labeler.yaml
```
Comment (takirala) [inline make/workflows.mk:4]: fancy :exploding_head:

**Review by takirala — APPROVED** [review]

File: make/workflows.mk (hunk 2 above)
Comment (takirala) [reply make/workflows.mk:4]: would be helpful if you can add a comment as to whats happening here in the code to make future edits easy :see_no_evil:

**Review by cbuto — APPROVED** [review]

**Review by bcmendoza — APPROVED** [review]

File: make/workflows.mk (hunk 2 above)
Comment (gracedo) [reply make/workflows.mk:10]: added in https://github.com/mesosphere/kommander-applications/pull/681/commits/59471402328e269e62b6cfe362dde5ae8c29a987

**Pull request merged**